experiment/run_experiment.py

```diff
@@ -157,21 +157,16 @@ def validate_fuzzer(fuzzer: str):
 
 def validate_fuzzer_config(fuzzer_config):
```

Comment:
I really want to get rid of command line fuzzer variants: https://github.com/google/fuzzbench/issues/292

Reply:
Will remove tomorrow.